packages/db/index.js

```diff
@@ -506,24 +506,40 @@ export class DBClient {
     }
   }
 
+  /**
+   * @typedef {Object} listUploadReturn
+   * @property {number} count - The total number of uploads for that user (used for pagination)
+   * @property {Promise<Array<import('./db-client-types').UploadItemOutput>>} uploads - The list of requested uploads.
+   */
+
   /**
    * List uploads of a given user.
    *
    * @param {number} userId
    * @param {import('./db-client-types').ListUploadsOptions} [opts]
-   * @returns {Promise<Array<import('./db-client-types').UploadItemOutput>>}
+   * @returns {listUploadReturn}
    */
   async listUploads (userId, opts = {}) {
+    const offset = opts.offset || 0
+    const limit = offset + (opts.size || 25)
+
+    const sortBy = opts.sortBy === 'Name'
```

Comment:
I wonder if it'd be better to raise if we try to sort on an unsupported column instead?